__tests__/getCategories.test.js

```diff
@@ -37,4 +37,40 @@ describe("app ", () => {
         });
     });
   });
+  describe("/api/reviews/:review_id", () => {
+    test("respond with a status of 200 and display a review object that match with ID endpoint", () => {
+      return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then((review) => {
+          expect(review.body[0].title).toBe(testData.reviewData[0].title);
+          expect(review.body[0].owner).toBe(testData.reviewData[0].owner);
+          //prettier-ignore
+          expect(review.body[0].review_body).toBe(testData.reviewData[0].review_body);
+          expect(review.body[0].designer).toBe(testData.reviewData[0].designer);
+          expect(review.body[0].category).toBe(testData.reviewData[0].category);
+          expect(review.body[0].votes).toBe(testData.reviewData[0].votes);
+          //prettier-ignore
+          expect(review.body[0].review_img_url).toBe(testData.reviewData[0].review_img_url);
+        });
+    });
+  });
+  test("a review object must have, review_id,title,review_body,designer,review_img_url,votes,category,owner,created_at - property", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then((review) => {
+        review.body.forEach((review) => {
```

Comment:
remember to do a length check before a forEach; however, I think this result too ought to be an object rather than an array